packages/sdk-react/src/components/payables/utils/isInvoiceOverdue.ts

```diff
@@ -0,0 +1,13 @@
+import { components } from '@/api';
+
+export const isInvoiceOverdue = (
+  invoice: components['schemas']['InvoiceResponsePayload']
+) => {
+  const today = new Date();
+  return (
+    // eslint-disable-next-line lingui/no-unlocalized-strings
+    ['new', 'pending', 'waiting to be paid'].includes(invoice.status) &&
```

Comment:
'new' is a wrong invoice status. I suggest to use typesafe approach:

```
  const statuses: components["schemas"]["ReceivablesStatusEnum"][] = ['draft', 'pending', 'waiting to be paid'];
  return (
    // eslint-disable-next-line lingui/no-unlocalized-strings
    statuses.includes(invoice.status) &&
```

Reply:
I took this array from Jira ticket description:

<img width="867" alt="Screenshot 2024-08-26 at 14 09 44" src="https://github.com/user-attachments/assets/ea64f182-0cb1-4776-9d50-eed4fe08da1e">

Reply:
You have to look not at Receivables, but at Payables enum

<img width="1133" alt="Screenshot 2024-08-26 at 14 33 16" src="https://github.com/user-attachments/assets/b209df16-7204-4329-89b8-c892876c0e2e">

Reply:
Discussed directly, the type is wrong, we have to change on PayablesEnum